examples/rich-text/package.json

```diff
@@ -15,14 +15,20 @@
   "license": "MIT",
   "dependencies": {
     "@teamwork/websocket-json-stream": "^2.0.0",
+    "bson-objectid": "^2.0.4",
     "express": "^4.18.2",
-    "quill": "^1.3.7",
+    "quill": "^2.0.2",
+    "quill-cursors": "^4.0.2",
     "reconnecting-websocket": "^4.4.0",
     "rich-text": "^4.1.0",
     "sharedb": "^3.3.0",
     "ws": "^8.12.1"
   },
   "devDependencies": {
-    "browserify": "^17.0.0"
+    "@babel/core": "^7.25.7",
+    "@babel/preset-env": "^7.25.7",
+    "babel-loader": "^9.2.1",
```

Comment:
Do we need these? I think if you remove `babel` from the Webpack config, everything still runs (in a modern browser). If possible, let's keep these examples as bare-bones as possible (it's not within the scope of this library to inform consumers how to set up their build chain).

Similar comment applies for `rich-text-presence`.